\documentclass[12pt,letterpaper]{article}
\usepackage{natbib}

%Packages
\usepackage{textcomp}
\usepackage{fullpage}
\usepackage{float}
\usepackage{latexsym}
\usepackage{url}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{mathtools}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsthm}
\usepackage{amstext}
\usepackage{enumerate}
\usepackage[osf]{mathpazo}
\usepackage{dcolumn}
\usepackage{lineno}
\usepackage{pdflscape}

\usepackage{color,soul}

\DeclarePairedDelimiter\abs{\lvert}{\rvert}%
\DeclarePairedDelimiter\norm{\lVert}{\rVert}%
\newcolumntype{d}[1]{D{.}{.}{#1}}

\pagenumbering{arabic}


%Pagination style and stuff
\linespread{2}
\raggedright
\setlength{\parindent}{0.5in}
% \setcounter{secnumdepth}{0} 
% \renewcommand{\section}[1]{%
% \bigskip
% \begin{center}
% \begin{Large}
% \normalfont\scshape #1
% \medskip
% \end{Large}
% \end{center}}
% \renewcommand{\subsection}[1]{%
% \bigskip
% \begin{center}
% \begin{large}
% \normalfont\itshape #1
% \end{large}
% \end{center}}
% \renewcommand{\subsubsection}[1]{%
% \vspace{2ex}
% \noindent
% \textit{#1.}---}
% \renewcommand{\tableofcontents}{}
%\bibpunct{(}{)}{;}{a}{}{,}

%---------------------------------------------
%
%       START
%
%---------------------------------------------

\begin{document}
%Running head
\begin{flushright}
Version dated: \today
\end{flushright}

\medskip
\begin{center}
\noindent{\Large \bf Disparities in the analysis of morphological disparity}
\bigskip

Thomas Guillerme$^{1,17,*, +}$,
Natalie Cooper$^{2, +}$,
Stephen L. Brusatte$^{3}$,
Katie E. Davis$^{4}$,
Andrew L Jackson$^{5}$,
Sylvain Gerber$^{6}$,
Anjali Goswami$^{2}$,
Kevin Healy$^{7}$,
Melanie J. Hopkins$^{8}$,
Marc E. H. Jones$^{9}$,
Graeme T. Lloyd$^{10}$,
Joseph E. O'Reilly$^{11}$, %was 17 - 11 forward + 1
Abi Pate$^{12}$,
Mark N. Puttick$^{13}$,
Emily Rayfield$^{12}$,
Erin E. Saupe$^{14}$,
Emma Sherratt$^{15}$,
Graham J. Slater$^{16}$,
Vera Weisbecker$^{1}$,
Gavin H. Thomas$^{17}$
and Philip C. J. Donoghue$^{12}$


\noindent { \it 
$^{1}$ School of Biological Sciences, University of Queensland, St. Lucia,
Queensland, Australia;
$^{2}$ Department of Life Sciences, Natural History Museum, London, Cromwell
Road, London, SW7 5BD, UK;
$^{3}$ School of GeoSciences, University of Edinburgh, Grant Institute,
Edinburgh EH9 3FE, UK;
$^{4}$ Leverhulme Centre for Anthropocene Biodiversity, Department of
Biology, University of York, YO10 5DD, UK;
$^{5}$ Department of Zoology, School of Natural Sciences, Trinity College
Dublin, Ireland;
$^{6}$ Institut de Systématique, Évolution, Biodiversité (ISYEB), Muséum
national d'Histoire naturelle, CNRS, Sorbonne Université, EPHE,
Université des Antilles, 57 rue Cuvier CP39, 75005 Paris, France;
$^{7}$ Ryan Institute, School of Natural Sciences, National University of
Ireland Galway, Ireland;
$^{8}$ Division of Paleontology (Invertebrates), American Museum of Natural
History, New York, USA;
$^{9}$ Department of Cell and Developmental Biology, University College
London, London, UK;
$^{10}$ School of Earth and Environment, University of Leeds, Leeds LS2 9JT,
UK;
$^{11}$ MRC Institute of Genetic and Molecular Medicine, University of Edinburgh, UK;
$^{12}$ School of Earth Sciences, University of Bristol, Life Sciences
Building, Tyndall Avenue, Bristol BS8 1TQ, UK;
$^{13}$ Milner Centre for Evolution, University of Bath, BA2 7AY UK;
$^{14}$ Department of Earth Sciences, University of Oxford, S Parks Road,
Oxford OX1 3AN, UK;
$^{15}$ School of Biological Sciences, The University of Adelaide, Adelaide,
South Australia 5005, Australia.
$^{16}$ Department of the Geophysical Sciences, University of Chicago,
Chicago, Illinois 60637;
$^{17}$ Department of Animal and Plant Sciences, The University of Sheffield,
Sheffield S10 2TN, UK;\\
$^{+}$ These authors contributed equally to the manuscript; $^{*}$ Corresponding
author: guillert@tcd.ie\\}


\end{center}

\modulolinenumbers[1]
\linenumbers

\textbf{Abstract (200 words max)}

\noindent Analyses of morphological disparity have been used to characterise and investigate the evolution of variation in the anatomy, function, and ecology of organisms since the 1980s.
While a diversity of methods have been employed, it is unclear whether they provide equivalent insights.
Here we review the most commonly used approaches for characterising and analysing morphological disparity, all of which have associated limitations that, if ignored, can lead to misinterpretation.
We provide best practice guidelines for disparity analyses, while noting that there can be no ``one-size-fits-all'' approach.
The available tools should always be used in the context of a specific biological question that will determine data and method selection at every stage of the analysis.
 
\textbf{Keywords}: multidimensionality, palaeobiology, ecology, morphology, disparity, variance/variation

\section{Introduction}

\noindent Clades of organisms are characterised by variation in both numbers of species and range of phenotypes through time.
At the extremes, clades may be exceptionally rich in species and phenotypic diversity (hereafter \textit{disparity}) (e.g. cichlids or molluscs), species-rich but disparity-poor (e.g. rodents or nematodes), species-poor but rich in disparity (e.g. afrotherian mammals), or depauperate in both species diversity and disparity (e.g. lungfish).
These phenomena suggest that taxonomic diversity and phenotypic disparity are not inextricably linked, raising important questions, such as: How does disparity evolve? Are some morphologies more common than others? Is anatomical evolution unbounded or are some anatomies impossible to achieve? What role does ecology play in structuring disparity? Analyses of species diversity have a venerable history, but those of disparity are comparatively more recent.
Originally defined as ``multidimensional morphological dissimilarity at a macroevolutionary scale'' \citep{runnegar1987rates,Gould1991}, the concept of disparity emerged from attempts by palaeobiologists to characterise the evolutionary origin of animal bodyplans and from attempts by comparative developmental biologists to provide causal explanations for their emergence.
However, disparity analyses have since expanded into comparative biology as a means of capturing how intrinsic and extrinsic causal agents affect morphological evolution.
Typically, methods to capture disparity are based on multidimensional spaces where each dimension represents an aspect of morphological variation (a trait) and biological observations (e.g. taxa) can be placed in this space based on their trait values.
Such multidimensional spaces (or morphospaces - defined broadly hereafter as a mathematical space relating morphological configurations generally based on some measure of similarity \citep{mitteroecker2009concept}) can then be used to tackle a diverse array of questions that can be grouped into four main (non-mutually exclusive) classes (Fig. 1): %remove fig1?


\begin{enumerate}

	\item \textbf{Descriptive disparity.} Pioneering studies of disparity characterised the shapes of organisms and how they differed among groups \citep{Foote1995, Briggs1992}.
	These studies described multidimensional patterns in morphological trait diversity by addressing pertinent questions: why are some morphological trait combinations more common than others, and what are the biological (or mathematical) properties of the resulting morphospace? \citep{Foote1995, Raup1961, Gerber2017}.
	More recently, this approach has been used to understand the relationship between developmental processes and morphology in the field of evolutionary development (evo-devo).
	For example, patterns of disparity have been used successfully to compare modules of evolution in various groups \citep{goswami2010influence,bardua2019morphological}, allowing researchers to link variation in shape to a group's evolutionary or developmental constraints \citep{Hipsley2017}.

	\item \textbf{Disparity-through-time.} This approach investigates how the morphologies of organisms have changed over time, by focussing on the disparity of taxa in particular time intervals or slices.
	This approach has been used widely in palaeobiology to answer a range of macroevolutionary questions, such as: how does disparity accumulate over the history of a clade \citep{prentice2011evolution, Guillerme2018}, or how does disparity change up to and across mass extinction events\citep{Friedman2010}?

	\item \textbf{Disparity and taxonomic diversity.} Morphological disparity provides another perspective on biodiversity; high morphological disparity represents a high diversity of morphologies (i.e.
	shapes or body plans) and is, presumably, associated with high levels of ecological and functional diversity \citep[but see][]{anderson2012}.
	This makes disparity an informative complement to diversity measures based on species richness alone.
	Indeed, most studies that have investigated disparity and taxonomic diversity support an effective decoupling of the two \citep[e.g.][]{Fortey1996, Hopkins2013}..
	The approach has been used to investigate whether some groups are more successful than others in their exploration of new evolutionary strategies \citep{pierce2008patterns}.

	\item \textbf{Disparity as a proxy for ecology.} The disparity of a group can be used as a proxy for either the functional role it plays within an ecosystem or its ecological niche.
	This approach assumes that groups with high disparity are also likely to be functionally and ecologically diverse, and that groups found in similar regions of shape space will have similar functional and ecological roles \citep{Pierce2008, Friedman2010}.
	The links between form and function, however, are not always clear.
	Traits can be linked to multiple functions and multiple functions can be linked to a single trait \citep{Wainwright2005}.
	This approach has been used to investigate hypotheses of competitive replacement \citep{tyler2011detecting} and changes in ecosystem function during and after mass extinctions \citep{Friedman2010}.
	It is particularly common in palaeobiology where it is not possible to directly observe many ecological or functional characteristics of extinct species \citep{Wainwright2005}.

\end{enumerate}

Fundamental insights into evolutionary biology have been elicited from these four types of disparity analysis.
One of the most important insights is the discovery that morphological disparity is often greatest early in the evolutionary history of clades \citep{Foote1997, Erwin2007, Hughes2013}, indicating that capacity for evolutionary innovation wanes as clade age, which some have argued reflects the evolutionary assembly of gene regulatory networks that constrain later fundamental change \citep{Erwin2007, Hughes2013}.
However, this example also highlights one of the greatest challenges confronting researchers who are attempting, increasingly, to obtain general insights from multiple independent studies: can the insights gained from studies using a diversity of methods, approaches and data types be considered equivalent?

In attempting to answer this question, we review current methods and highlight their limitations, as part of a more general attempt to propose best practice guidelines for studies of disparity.
We first discuss the appropriate data required for characterising disparity, then review various challenging aspects of these approaches.
Throughout, it is important to remember that these tools should always be used in the context of a specific scientific question, as this will drive data and methodological choices at every stage of the process.

% \begin{figure}[!htbp]
% \centering
%    \includegraphics[width=0.9\textwidth]{Figures/figure_disparities.pdf}
% \caption{
%     The four main types of disparity analysis. \textbf{Descriptive disparity} focuses on describing the features of morphospace occupation; \textbf{disparity-through-time} investigates the evolution of the morphospace through time including the effect of extinction events; \textbf{disparity and taxonomic diversity} compares different measures of biodiversity; \textbf{disparity as a proxy for ecology} uses disparity as a proxy for the ecological or functional role of a group.
%     These categories are not independent and many disparity studies will cover more than one.
% }
% \label{Fig:disparity}
% \end{figure}



\section{Data and disparity} \label{section:data}


\noindent Disparity analyses are based on traits, but traits can be characterised in a number of ways:
1) discrete morphological characters, e.g. coding the absence or presence of features or a discrete characteristic of a trait \citep[e.g.][]{Foote1989, Deline2018};
2) continuous measurements of features \citep[e.g. lengths in][]{Friedman2010};
or 3) more mathematical descriptors from geometric morphometric landmark data (e.g.
Procrustes coordinates)  \citep[e.g.][]{sherratt2017rates}, Fourier coefficients \citep[e.g.][]{Foote1995, Spriggs2018} or model-based descriptors \citep[e.g.][]{Raup1961,Saunders2004} (Fig. 2).
None of these approaches is superior, but they may be more or less well-suited to characterising the traits compared and to the question being asked using those traits \citep{hetherington2015cladistic,Hopkins2017}.

For example, if investigating variation of bat wing shapes, both homologous landmarks and continuous measurements of bones may be appropriate to capture patterns of wing variation.
If the question focuses on comparing wings between bats and birds, however, different measurements might be more appropriate depending on the specific question.
That is, if the focus is whether the aerodynamic properties of wings vary within bats or between bats and birds, the traits collected should reflect these aerodynamic properties (e.g. wingspan, aspect ratio, etc.).
However, if the focus is on convergence between different bats and birds, it would be preferable to use traits that have facilitated flight in both groups (e.g.
digit length, integumentary system, etc.).
Where there is any doubt about which traits to analyse, it may be preferable to use several different kinds of data for the same feature to determine whether they capture the same pattern of disparity.

The points above assume that researchers are collecting their own data for disparity analyses, but this is often not the case.
Discrete characters are commonly recycled from phylogenetic studies \citep[e.g.][]{Brusatte2008,Close2015}.
This is an efficient approach to character sampling, but it may artifactually increase disparity between phylogenetically distinct groups, since phylogenetic characters are often collected to discriminate among groups.
This needs to be considered when interpreting results, especially as synapomorphies will naturally lead to an apparent shift or increase in disparity when new clades appear.
Furthermore, many datasets are limited to subsets of anatomy that are at least implicit samples of overall anatomy, but explicit tests of this assumption have shown that different aspects of morphology can exhibit different patterns of disparity \citep{Hopkins2017}.
The influence of trait choice on resulting disparity patterns can be especially challenging where the available data has non-random missing anatomical parts, such as the absence of soft tissue in the fossil record \citep{Deline2018}.

Trait data suffer from the same shortcomings as most biological datasets -- data can be missing, non-overlapping, hierarchical, inapplicable, ambiguous, polymorphic, correlated, or there may be an insufficient sample size \citep{Palci2018}.
Biological phenomena such as allometry and sexual dimorphism may also influence trait data.
More mundanely, data collection is constrained by the time and money available, making collating a ``perfect'' dataset impossible.
Ultimately, disparity analyses are characterised by the data, and subsamples of the universe of possible data may not have the power to uncover the full patterns of disparity.
Therefore, trait data should be collected with the scientific question in mind, or the question asked should be tailored to the limits of the data available.

\begin{figure}[!htbp]
\centering
   \includegraphics[width=0.9\textwidth]{Figures/figure_data.pdf}
\caption{
   Major routes to obtain morphological data for disparity analyses. Data can be collected as discrete trait observations (e.g. presence-absence data) or as continuous data. 
   Continuous data can be collected by various methods including linear measurements and landmark coordinates or contours (curves).
   These measurements can then be mathematically transformed (logarithmic transformations, scaling, Procrustes superimposition, elliptic Fourier transforms, etc.).
   Regardless of the method, data collection produces a trait matrix where the observed traits constitute columns and the studied elements (generally taxa or OTUs) the rows. 
}
\label{Fig:data}
\end{figure}

\section{Disparity analysis methods} \label{section:methods}

\noindent Once suitable trait data have been collected, the design of the disparity analysis itself needs to be considered.
Study design encompasses several key aspects including \ref{section:ordination} the difficulty of dealing with multidimensional data; \ref{section:metrics} the indices used to summarise the relative disparity of groups; \ref{section:testing} the methods used for hypothesis testing within the disparity analysis framework; and \ref{section:phylo} the influence of phylogeny on disparity analyses.
We consider these aspects in order below.

\subsection{To ordinate or not to ordinate - that is the (multidimensional) question} \label{section:ordination}

Disparity analyses often use ordination techniques for dimension reduction.
Ordinations are statistical methods that map observed variables onto a new space of reduced dimension while maintaining the requirement that similar observations are closer together than dissimilar ones (e.g. principal component analysis - PCA; principal coordinates analysis - PCO/PCoA; non-metric multidimensional scaling - NMDS; etc.).
They come in many flavours depending on the data and the desired morphospace properties.
For example, quantitative (continuous) data can be reduced using PCA, and qualitative data (or mixed data types) can be reduced using PCO (which is equivalent to metric multidimensional scaling (MDS)) or non-metric multidimensional scaling \citep[NMDS; see][chapter 9 for a detailed overview of ordination methods and properties]{Legendre2012}.

One of the reasons why ordination techniques are common in disparity analysis is that they make it easier for researchers to comprehend patterns in two or three spatial dimensions at a time, which can be more intuitive than through disparity indices (see section \ref{section:metrics} below).
Additionally, after ordinating the data, it is possible to focus on just a subset of axes of the morphospace (i.e.
selecting only those axes that describe the majority of the variation in the dataset - e.g. 95\%).
In the case of geometric morphometric data, ordination is particularly useful as it conserves the mathematical properties of the data while efficiently reducing the dimensions \citep{Legendre2012,dryden2016statistical}.
In practice, this facilitates interpretation of only the major axis of a highly dimensional dataset as major gradients of biological variation \citep[e.g. the elongation and flattening of birds beaks;][]{bright2016shapes}.

Like most other aspects of disparity analyses, however, reducing dimensionality can be fraught.
In the case of ordination, subsampling axes from the ordination can lead to misinterpretation of the results.
Although a common technique is to consider the \textit{d} axes that encompasses 95\% or 99\% of the variance in the dataset, the interpretation of these principal axes can be unrepresentative of the data and lead to misinterpretation of the biological variation mapped on these axes \citep{Bookstein2015, Weisbecker2019}.
Visual interpretations of multidimensional data can be particularly misleading, not least as multidimensional spaces are not necessarily Euclidean even when analysing morphometric data \citep{Deline2018, Gerber2017}.

Interpreting biological variation along the axes is always a \textit{post-hoc} procedure and may have little relation to the overall question (for example, if the first few ordination axes represent elongation of the beak in birds, but the question is about wing disparity).
Additionally, in some cases, reducing the dimensionality of a dataset can render its interpretation more problematic.
For example, when the analysed data are non-Euclidean (e.g. some types of discrete morphological characters such as inapplicable characters), interpreting the resulting ordinated space can be challenging \citep{Gerber2019}.
This is problematic when comparing the position of groups in multidimensional space, as the distances might not be linear.
Furthermore, \textit{post-hoc} interpretations of the gradient of variation on the ordination axes may be biologically meaningless or simply impossible \citep{Gerber2019}.
Although some gradients are easy to detect or interpret (e.g.
the elongation and depth of mandibles in fishes on first and second PC axes, respectively; \citealt{Hill2018}), some are not \citep[e.g.][]{Weisbecker2019}.
For example, with discrete morphological data, a gradient between the species that have many characters in state 1 and those that have more in state 0 has no biological meaning if these are binary alternate states.

In general, categorical data are a good deal more problematic than continuous data, because the characters themselves are invariably non-equivalent, non-independent, and mostly non-Euclidean, and the distribution of the variance is usually more evenly distributed across axes (i.e. contrary to a PCA, the first few axes do not encompass most variance of the dataset).
Such non-Euclidean spaces often have non-intuitive properties, for example, straight lines viewed in bivariate plots of some dimensions are not actually straight and, character coding and missing data can make the pairwise dissimilarity matrix lose its metric properties \citep[i.e. the distance between A and B is not equal to the distance between B and A;][]{Gerber2014}.
Last, but not least, in many cases, ordination might not be necessary.
For example, if an index characterising disparity can use all of the data, it is not necessary to calculate it on the ordinated dataset \citep[e.g.][]{Close2015}.
For all of these reasons, multidimensional data should not be ordinated automatically, and careful consideration should be given to whether the aim of the study can be achieved without ordination \citep{lloyd2016,lloyd2018}.

\begin{figure}[!htbp]
\centering
   \includegraphics[width=0.9\textwidth]{Figures/figure_space.pdf}
\caption{
    \tiny{
    \textbf{Morphospaces}: different mathematical representations of a morphospace. A trait matrix can be transformed into a distance matrix \citep[e.g in][]{Close2015} or an ordinated matrix \citep[e.g. in][]{tyler2011detecting}.
    Here we consider all these matrices as being \textit{morphospaces}, i.e. objects containing all the combinations of traits and observations (albeit transformed differently).
    \textbf{Visualisation}: different  ways to represent the morphospace in 2D.
    Visualisations can use either trait plots (directly from the trait matrix); ``flattening'' ordination of the data (e.g. using non-metric multidimensional scaling forcing the dataset into two dimensions - note that variance is lost due to the distortion stress and non-metric properties of the resulting operation); or ordination axis plots (directly from the ordinated matrix).
    Note that in 2D representations, it is good practice to plot both axes on the same scale to avoid visually distorting the importance of one axis). Data depicted is from the classic ``iris'' dataset of \citep{edgar1935irises,fisher1936use}.
    }
}
\label{Fig:morphospace}
\end{figure}

\subsection{Measuring disparity} \label{section:metrics}

Most disparity datasets are multidimensional and, consequently, a large component of any disparity analysis involves considering how to extract a meaningful (i.e.
interpretable) summary of disparity.
This is usually achieved with a disparity measure or index \citep{Hopkins2017}.
As with any summary of multidimensional data, disparity indices will reflect only some aspects of the morphological variation, never its whole complexity \citep{GuillermeMOMS}.
It is therefore often beneficial to use more than one index to summarise different aspects of variation, guided by the aim of the study.

When considering only one dimension, disparity indices can be used to reflect the spread of the distribution (e.g.
the range, quantiles or variance) or its central tendency (i.e., mean, median or mode).
Among these indices, some will have more attractive properties than others, such as sensitivity to outliers.
Range and mean are highly sensitive, whereas quantiles, variance and median are less so, making them more or less appropriate for different questions.
For example, if the goal is to characterise the extent of morphospace occupied by a group (e.g. does group A occupy as much space as group B?), indices related to the spread of the group in the morphospace are most appropriate (e.g.
volume; \citealt{Diaz2016}, distance from the centroid; \citealt{Hopkins2017, Finlay2015}, variance and range; \citealt{Brusatte2008}).
Furthermore, aspects other than variation (\textit{sensu} disparity) can be of interest: if we wish to describe the ``position'' of a group in a morphospace (e.g.
does group A occupy the same space as group B?), indices related to the distance between the elements within a group and a fixed point in the morphospace are most appropriate \citep{GuillermeMOMS}.
Finally, if we aim to characterise the density of morphospace occupation (e.g.
is group A more closely packed than group B?) indices related to the pairwise distances between elements will be most appropriate (e.g. nearest neighbour distance, pairwise distances, etc. \citealt{Close2015} - see section \ref{section:ordination} above).

In addition to considering which properties of disparity these indices capture, it is also important to consider the mathematical properties of the indices and their associated caveats \citep{Wills2001, Ciampaglio2001}.
For example, measuring the sum of variance for each dimension of the space is equivalent to measuring the sum of variance in a PCA (because the co-variance is null).
However, this is no longer true of other transformations of the space or when a subset of dimensions or elements are considered, such as is often done after PCA \citep{Legendre2012}.

Furthermore, multidimensional spaces have some counterintuitive properties that should be considered, such as the ``curse of dimensionality'' \citep{Bellman1966}.
In spaces with some axes of variance lower than one, product-based indices used as proxies of volumes (e.g. product of ranges, hypervolume, hypercube, etc.) can tend quickly tend towards zero for spaces with even a modest number of dimensions \citep{Bellman1966}.
Other types of indices are also extremely sensitive to outliers and can be biased easily by sample size, for example range \citep{Wills2001} or convex hull based \citep{Jackson2011} indices.

\subsection{Testing hypotheses about the evolution of disparity} \label{section:testing}

No matter which disparity indices have been calculated, the research question must be framed in an appropriate statistical context.
The multidimensional statistical toolkit for ecology and evolution has greatly expanded in recent years \citep{clavel2015mvmorph, Adams2018}, but some of these advances have yet to be implemented in disparity analyses.
Instead, hypothesis testing has been mostly confined to a small set of well-established methods.
One commonly used test is the non-parametric permutation analysis of variance (PERMANOVA) \citep{Anderson2001} that tests whether two groups share the same centroid and dispersion based on a distance matrix between observations.
This test therefore does not require the morphospace to be ordinated (see section \ref{section:ordination} above).
Statistical tests should be employed that are tailored to the question at hand, rather than simply following common practices.

It is also important to consider which data should be subjected to a statistical test.
For example, in morphological disparity analysis, especially for palaeobiological questions, data are often bootstrapped.
This has two advantages: (i) when the disparity index is unidimensional (e.g.
the sum of variances), bootstrapping the data generates a distribution of the index that can be analysed using the vast statistical toolkit available for comparing distributions; (ii) when data are scarce, bootstrapping the data allows users to introduce variance, rendering the test less sensitive to outliers.
However, bootstrapped data are pseudoreplicates and thus non-independent.
Furthermore, the number of bootstrap pseudoreplicates will inevitably increase the false positive rate (Type I error) \citep{strube1988bootstrap}.
These factors are often ignored in disparity analyses.

Finally, it is important to understand the limitations of the dataset for performing statistical analysis.
Mainly, disparity analysis should be restrained to groups within the same morphospace and are more difficult between different morphospaces.
This can be the case when comparing elements with different numbers of landmarks or different landmark configurations which will result in different morphospaces;comparing disparity indices between these is not trivial.

\subsection{Disparity and phylogeny} \label{section:phylo}

As with all comparative datasets, the data used in disparity analyses are not independent because close relatives will tend to have more similar morphologies than more distant relatives \citep{Harvey1998}.
Thus, for disparity analyses that consider groups with phylogenetic relationships (which is common) the non-independence between observations should be taken into account.
It has been noted, however, that some popular phylogenetic correction methods (like phylogenetic PCA) can be inappropriate, especially when using only the first $n$ axis of the ordination, and can lead to incorrect interpretations of the data \citep[such as wrongly supporting ``early burst'' type patterns;][]{Uyeda2015}.

One other common way to take phylogeny into account in disparity analyses is using ancestral state estimations in disparity-through-time analyses to extract disparity estimates for non-sampled taxa and/or nodes of a phylogeny.
Ancestral state estimation can be performed at two points in the disparity analysis pipeline: either (1) pre-transformation, i.e. the estimation is done before transformation of the data (e.g. ordination, or distance matrix construction) and is simply based on the original data; or (2) post-transformation, i.e. the estimation is done after transformation of the data by estimating the ancestral states using the transformed matrix \citep[e.g. the ordination scores;][]{lloyd2018}.

Pre-transformation ancestral state estimation will change the way the ordination space is defined -- i.e. the relationship between the points are not yet estimated -- and requires longer computational times.
However, once the morphospace is defined its properties will not change.
Post-transformation ancestral state estimation will not change the empirical ordination space and is faster to compute, but it will add elements in the space, whose estimated positions can be problematic for statistical tests and evolutionary inferences down the line \citep{lloyd2018}.

All ancestral state estimates are highly dependent on the data and method used \cite{Louca2019}.
In general, using ancestral state estimation can help with recovering patterns of changes in disparity but should not be used simply to generate extra data points to increase statistical power.
In fact, these extra points are not independent and can also have problematic side effects, especially when testing for the influence of mass extinctions on disparity as they artificially increase taxon sampling.

\section{Disparity analyses for the future} \label{section:future}

\noindent Morphological disparity analyses are widely employed in evolutionary palaeobiology, and are based on a diversity of methods and data.
There is no ``one-size-fits-all'' pipeline for morphological disparity analyses.
As with any multidimensional analysis, there are many variables that have to be considered when deciding which data to use and how to analyse them, stemming from the explicit hypotheses being tested.
Many of the problems in morphological disparity analysis arise from ``blind'' application of established methodological pipelines without consideration of the biological question being addressed.
We advocate that researchers should assemble their analytical approaches based on an experimental approach that explores the impact of competing methods, such as choice of indices, ordination method and ancestral state estimation method on disparity analysis results.
Thankfully this is becoming easier through the availability of diverse well documented R packages for multidimensional analysis \citep{Bouxin2005, oksanen2007vegan, Harmon2008, lloyd2016, Guillerme2018b}.
Many of the methods employed in disparity analysis are used more widely in other fields, including genomics and ecology, which also encompass analyses of multidimensional datasets \citep{Donohue2013, Saupe2015, Canter2018, mammola2019}.
Innovations in morphological disparity analyses likely await discovery in their respective literatures.

While studies of morphological disparity would benefit from advances in multidimensional analysis in other fields, the concept of a morphospace could reciprocally benefit other disciplines.
For example, the multidimensional analysis of \citep{Diaz2016}, which analysed patterns of form and function in plants, is essentially an eco-morphospace; isotopic analyses of organisms \citep{Jackson2011,Swanson2015} can be represented as an isotope-space; ecosystem functioning in \citealt{Donohue2013} as an ecosystem-space, etc.
These generalisations could also be exported for any set of traits: cognate approaches have been adopted in the analysis of single cell comparative transcriptome data \citep{Sebe-Pedros2018} where interpretation of the resulting transcriptome-spaces would be improved by giving careful attention to the concerns we highlight concerning morphospaces.

Although disparity analyses are now simple to implement in freely available software \citep{Bouxin2005, oksanen2007vegan, Harmon2008, lloyd2016, Guillerme2018b} it is crucial to remember that they are multidimensional analyses and that multidimensional analyses are complex.
We assert that future morphological analyses will benefit from emphasising the methodological decisions made, rather than simply using disparity analysis because it exists.

\subsection{Author contributions}

TG, NC and PD proposed this review; TG and NC led the writing supported by PD and GT. All authors edited drafts and approved the final version.

\subsection{Acknowledgements}

This article results from discussion at the Royal Society International Science Seminar on `Reconciling disparate views on disparity' held at Chicheley Hall, 9-10th January, 2018.
AG was funded by European Research Council Starting grant 637171 ADaPTiVE;
ALJ was funded by an Irish Research Council Laureate Award IRCLA/2017/186;
ES was funded by a University of Adelaide Research Fellowship;
EES was funded by a Leverhulme Trust Research Project Grant (DGR01020);
PD was funded by NERC (NE/P013678/1; NE/N002067/1) and BBSRC (BB/N000919/1);
SB was funded by European Research Council (ERC) under the European Union's Horizon 2020 research and innovation program (grant agreement No 756226, ERC Starting Grant: PalM) and a Leverhulme Trust Research Project Grant (RPG-2017-167);
TG was funded by ARC DP170103227 and FT180100634 awarded to VW.


\subsection{Appendix: morphospace definition in this paper} \label{section:appendixmorphospace}

\noindent \cite{mitteroecker2009concept} define a morphospace as:

\setlength{\leftskip}{1cm}
\setlength{\rightskip}{1cm}

\noindent \textit{``[a] mathematical spaces describing and relating the phenotypic configuration of biological organisms [...]. In a typical morphospace, the morphological configuration of an organism is represented by a single point, and the dimensionality of the space is determined by the number of measured variables (Q-space).''} \citealt{mitteroecker2009concept} p.55.

\setlength{\leftskip}{0pt}
\setlength{\rightskip}{0pt}

\noindent Here, we generalise the definition of a morphospace following \cite{mitteroecker2009concept} as the Q-space or any of its linear transformation.
The morphospace is any matrix $(a_{ij})$ with $n$ observations with $d$ trait values each or any linear transformation $f: V \to W$ of $(a_{ij})$.
In other words the morphospace is the observed data matrix (e.g. the collected data) or any of it's linear transformation (e.g. a PCA, a distance matrix, a log, etc...).
In \cite{mitteroecker2009concept}'s definition, the morphospace needs a conserved $n \times d$ dimensions whereas our definition propose only the conservation of $n$ dimensions (the number of observations) as long as the morphospace can be expressed as:

\begin{equation}
    M1 = f(a_{ij})\in \mathbb{R}^{{n}\times{d}}
\end{equation}

\noindent Where $f$ can be any linear transformation function (including no transformation: $f(x) = x$).
This generalisation allows to consider linear transformations of $(a_{ij})$ as a different ``orientations'' or ``resolutions'' (not in the mathematical sense \textit{per se}) of the observed trait matrix $(a_{ij})$. For example, a distance matrix of $(a_{ij})$ can be represented through the following linear transformation:

\begin{equation}
    M2 = f_{a_{ij}} = D_{a_{ij}}
\end{equation}

\noindent Where $D$ is a distance function for any pair of $ij$ in the matrix (e.g. if the distance is Euclidean $D_{ij} = \sqrt{\sum(i_{d}-j_{d})^2}$).
$M2$ is then just a different representation of the original morphospace $(a_{ij})$ where the properties between the $n$ observations are expressed as distances.
Note that in our definition, $f_{a_{ij}}$ does not need to result in a metric (or metric Euclidean) space although this is often desired in disparity analysis.

Our definition doesn't consider a matrix to be a morphospace of $(a_{ij})$ if it results from a non-linear transformation of $(a_{ij})$ matrix.
For example if one performs an ordination (e.g. PCA) on the matrix $(a_{ij})$ such as:

\begin{equation}
    M3 = f_{a_{ij}} = P(a_{ij})
\end{equation}

\noindent Where $P$ is a principal components linear transformation.
$M3$ is still a morphospace (i.e. a linear transformation of $(a_{ij})$). However if one only uses the two first axis of $M3$, this would result in a different space because it is no more a linear transformation of $(a_{ij})$.

However, we \textbf{\underline{\textit{DO NOT}}} declare that doing such a non-linear transformation is incorrect in a disparity analysis context.
As pointed out throughout the main manuscript, different methods can be appropriate for different questions and different datasets.
The point of this appendix is to develop \textit{our} definition of a morphospace but we do not argue that any other definition is incorrect.
We do, however, suggest that a definition of ``morphospace'' should be at least briefly noted in papers using this mathematical or biological concept.


\bibliographystyle{sysbio}
\bibliography{References}

\end{document}

